docs/CONTRIBUTING.md

```diff
@@ -1 +1,153 @@
+$ git clone git@github.com:xxx/github-analysis-report.git
+$ cd github-analysis-report
+```
+
+### 4. Create a new branch
+
+Create a new branch for development.
+
+```shell
+$ git checkout -b your-branch-name
+```
+
+The name of branch should be semantic, avoiding words like `update` or `tmp`. We suggest to use `feature/xxx` if the modification is about to implement a new feature.
+
+### 5. Commit your changes
+
+Now you can create your patch or add `SQL` statement in the new branch, and commit your changes by:
+
+```shell
+$ git add .
+$ git commit -sm "docs: add workflow doc (#26)"
+```
+
+You are encouraged to use [angular commit-message-format][angular-commit-message-format] to write commit message. In this way, we could have a more trackable history and an automatically generated changelog.
+
+### 6. Sync your local repository with the upstream
+
+Keep your local repository updated with upstream repository by:
+
+```shell
+$ git remote add upstream git@github.com:X-lab2017/github-analysis-report.git
+$ git fetch upstream master
+$ git rebase upstream/master
+```
+
+If conflicts arise, you need to resolve the conflicts manually, then:
+
+```shell
+$ git add my-fix-file
+$ git rebase --continue
+```
+
+### 7. Push your branch to GitHub
+
+```shell
+$ git push -f origin your-branch-name
+```
+
+### 8. Create a Pull Request
+
+In GitHub, send a pull request to [`X-lab2017/github-analysis-report`][repo].
+
+Please sign our [Contributor License Agreement (CLA)](#cla) before sending PRs.
+
+The core team is monitoring for pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation.
+
+If we suggest changes then:
+
+-   Make the required updates.
+
+-   Commit your changes with `--amend` and force push to your GitHub repository (this will update your Pull Request):
+
+    ```shell
+    $ git add .
+    $ git commit --amend
+    $ git push -f origin branch-name
+    ```
+
+That's it! Thank you for your contribution!
+
+You can refer to [workflow](https://www.x-lab.info/github-analysis-report/#/workflow?id=appendix) to see more information about the `PR` workflow with `SQL` files invovled.
+
+### 9. After your pull request is merged
+
+After your pull request is merged, you can safely delete your branch and pull the changes from the upstream repository:
+
+-   Delete the remote branch on GitHub either through the GitHub web UI or your local shell as follows:
+
+    ```shell
+    $ git push origin --delete your-branch-name
+    ```
+
+-   Check out the master branch:
+
+    ```shell
+    $ git checkout master -f
+    ```
+
+-   Delete the local branch:
+
+    ```shell
+    $ git branch -D your-branch-name
+    ```
+
+-   Update your master with the latest upstream version:
+
+    ```shell
+    $ git pull --ff upstream master
+    ```
+
+## <a name="cla"></a> Signing the CLA
+
+Please sign our [Contributor License Agreement (CLA)][cla] before sending pull requests. For any code
```

Comment:
We can remove the CLA part because we do not have CLA now and I don't think we need CLA for this project.

Reply:
ok